### PR TITLE
Consolidate ci jobs into 1

### PR DESCRIPTION
**Pull request by estherkim**

### CODE REVIEW

File: configs/versions.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "experimentB": null,
   "experimentC": null,
   "lts": "012201122239000",
-  "nightly": "042201212122000",
+  "nightly": "042201212122001",
```
Comment (danielrozenberg) [inline configs/versions.json:11]: Undo this change

Comment (estherkim) [reply configs/versions.json:11]: still testing :)

**Comment by estherkim** [comment]
Will change branch protection rule to require 'tests' instead of 'config' and 'setup' after force merging this.

**Comment by danielrozenberg** [comment]
> Will change branch protection rule to require 'tests' instead of 'config' and 'setup' after force merging this.

Oh lol I changed the branch protection rules so you wouldn't have to force-merge this :D
